### DISCUSSION TITLE
Issue regarding Upadation of readme having correct setup procedure.

### DISCUSSION THREAD

**Discussion by MuKulsoop** [recodehive/awesome-github-profiles#677] [Ideas]
As the code is in the gh-pages branch. Can a issue be raised regarding updating the readme file having proper steps for the setup of the project in the local environment. 
Like using command "git clone --branch gh-pages https://github.com/Username/awesome-github-profiles.git" to clone the code in the local environment ( for beginners ).

**Comment by sanjay-kv** [comment]
https://github.com/recodehive/awesome-github-profiles/issues/683

**Comment by sanjay-kv** [comment]
you can work on it
